src/components/buttons/TintButton.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { ReactNode } from 'react';
 import { ButtonProps } from 'react-native';
-import { TextColor } from '../../design-system/typography/typography';
+import { TextColor } from '../../design-system/color/palettes';
```

Comment:
This `TextColor` type was duplicated between `typography.ts` and `palettes.ts`, so this PR aligns on the latter, deleting color-related code from `typography.ts`.